packages/ProgressIndicators/src/ProgressBar/index.tsx

```diff
@@ -10,81 +10,90 @@ export interface ProgressBarProps {
   value: number /** Represents the progress bar value */;
   lineColor: string /** set line colors */;
   lineColorThresholds?: { [key: string]: number } /** set linecolor threshold */;
-  cssClass: string /** sets either stripped or animated css gradient over progressBar */;
+  stripedCssClass: string /** sets stripped css gradient over progressBar */;
```

Comment:
Do you anticipate that this will only be used to make the progress bar striped?  It seems to me that this prop supplies CSS classes to the component, and this may be `progress-bar-striped` or any other. If so then I would keep the old name that is more generic i.e. `cssClass`

Reply:
Agreed